core/server/data/migrations/versions/3.12/01-remove-legacy-is-paid-flag-from-settings.js

```diff
@@ -0,0 +1,110 @@
+const _ = require('lodash');
+const common = require('../../../../lib/common');
+const debug = require('ghost-ignition').debug('migrations');
+
+module.exports.config = {
+    transaction: true
+};
+
+module.exports.up = (options) => {
+    let localOptions = _.merge({
+        context: {internal: true}
```

Comment:
What is this for? We're not interacting with models so shouldn't need it :thinking:

Reply:
Yup. Good one to drop. Have to get the copy-pasting habit for migrations under control :smile: